.github/workflows/release_asan_instrumentation_clang.yaml

```diff
@@ -0,0 +1,119 @@
+name: release_asan_instrumentation_clang
```

Comment:
Why do we need two additional workflows instead of adding new jobs to `.github/workflows/debug_asan_clang.yml` and `.github/workflows/release_asan_clang.yml`?